packages/contracts/test/voting/multisig.ts

```diff
@@ -502,6 +604,40 @@ describe('Multisig', function () {
       it('returns `true` if the approver is listed', async () => {
         expect(await multisig.canApprove(id, signers[0].address)).to.be.true;
       });
+
+      it("returns `false` if the proposal hasn't started yet", async () => {
+        await multisig.createProposal(
+          dummyMetadata,
+          dummyActions,
+          false,
+          false,
+          await timestampIn(2000),
+          await timestampIn(5000)
+        );
+
+        expect(await multisig.canApprove(1, signers[0].address)).to.be.false;
```

Comment:
Shouldn't this use the other `await expect()` shape?

Applies to the rest of places